clash-lib/src/Clash/Normalize.hs

```diff
@@ -19,6 +19,9 @@
 
 module Clash.Normalize where
 
+import Debug.Trace -- TODO
```

Comment:
TODO? :P

Reply:
I think we should have more random TODOs scattered in the code. After all, our work is never done :wink: